libshvvisu/src/logview/dlgloginspector.cpp

```diff
@@ -90,6 +90,45 @@ DlgLogInspector::DlgLogInspector(const QString &shv_path, QWidget *parent) :
 		ui->btSaveData->setMenu(m);
 	}
 
+	{
+		auto *m = new QMenu(this);
+		{
+			auto *a = new QAction(tr("ChainPack"), m);
+			connect(a, &QAction::triggered, this, [this]() {
+				const std::string &log_data = loadData(".chpk");
```

Comment:
use just `std::string` here, no reason for cref, because loadData doesn't return a reference